Template/Components/Frame.html

```diff
@@ -26,7 +28,9 @@
                 <div>
                     <p>[* Εθελοντισμός */* Volunteer *]</p>
                     <div>
-                    <a href="[*localization root*][* Προσωπικό.html */* Staff.html *]">[* Προσωπικό */* Staff *]</a>
+                    <a href="[*localization root*][* Προσωπικό.html */* Short-term Staff.html *]">[* Προσωπικό */* Short-term Staff *]</a>
```

Comment:
General proofreading: Two words joined with a hyphen should still both be capitalized. (You can fix the filename too if you want; it may appear in the address bar. I just cannot use GitHub’s suggestion interface to rename a file for you.)

```suggestion
                    <a href="[*localization root*][* Προσωπικό.html */* Short-term Staff.html *]">[* Προσωπικό */* Short‐Term Staff *]</a>
```